card/mod/01_core/spec/set/all/tracked_attributes_spec.rb

```diff
@@ -161,7 +161,7 @@ def cards_should_be_added number
       assert @b.errors[:name]
     end
 
-    it "test_update_dependents" do
+    it "test_update_descendants" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.